kyuubi-server/src/main/scala/org/apache/kyuubi/operation/ExecuteStatement.scala

```diff
@@ -86,7 +91,7 @@ class ExecuteStatement(
     var statusResp: TGetOperationStatusResp = null
     var currentAttempts = 0
 
-    def getOperationStatusWithRetry: Unit = {
+    def fetchOperationStatusWithRetry(): Unit = {
```

Comment:
what's wrong with get?

Reply:
IDEA complains "Accessor-like method has Unit result type"

Reply:
are you mixing java settings with scala?

Reply:
I'm using the default setting